testsuite/tests/readonly_property/test.py

```diff
@@ -0,0 +1,9 @@
+import gen
+
+p = gen.Point()
+
+print p.x
+try:
+    p.x = 12
+except Exception, e:
```

Comment:
If you want to have Python3 compatibilty, this should be: `except Exception as exc:`